tests/test_Coal_skyline/scripts/mcmc_Coal_skyline_fixed.Rev

```diff
@@ -41,9 +41,21 @@ for (i in 1:NUM_INTERVALS) {
 
 }
 
+num_events_pi <- rep(1, NUM_INTERVALS)
```

Comment:
Can you add a short comment about what this new code is doing?

Reply:
Done.